hatrac/model/storage/amazons3.py

```diff
@@ -32,19 +35,31 @@ def __init__(self, config):
         self.bucket_name = config['s3_bucket']
 
         # TODO: do something better for authz than storing secret in our config data!
-        self.conn_config = config['s3_connection']
```

Comment:
My intention here was to include a nested configuration object under the 's3_connection' key in the hatrac configuration JSON object.  It would be passed into the S3Connection class as keyword arguments so that we don't have to keep adjusting code here to support different custom parameters a deployment might require.